website/views.py

```diff
@@ -38,53 +85,147 @@ def register(request):
+# 	if request.user.is_authenticated:
+# 		if request.method == "POST":
+# 			if form.is_valid():
+# 				add_record = form.save()
+# 				messages.success(request, "Record Added...")
+# 				return redirect('home')
+# 	    return render(request, 'add_record.html', {'form':form})
+# 	else:
+# 		messages.success(request, "You Must Be Logged In...")
+# 		return redirect('home')
+# @login_required(login_url='home')
+# def add_record(request):
+#     if request.method == "POST":
+#         form = AddRecordForm(request.POST,request.FILES)
+#         if form.is_valid():
+#             add_record = form.save()
+#             messages.success(request, "Record Added...")
+#             return redirect('home')
+#     else:
+#         form = AddRecordForm()
+#
+#     return render(request, 'add_record.html', {'form': form})
+class add_record(LoginRequiredMixin,View):
+    template_name='add_record.html'
+    def post(self,request):
+        form = RecordForm(request.POST, request.FILES)
+        if form.is_valid():
+            add_record = form.save()
+            messages.success(request, "Record Added...")
+            return redirect('home')
+        else:
+            messages.success(request,"Invalid -- Fill the form Again!!")
+            form = RecordForm()
+            return render(request, self.template_name, {'form': form})
+    def get(self,request):
+        form = RecordForm()
+        return render(request, self.template_name, {'form': form})
+
+# Function based UpdateView
+# def update_record(request, pk):
+# 	if request.user.is_authenticated:
+# 		current_record = Record.objects.get(id=pk)
+# 		form = AddRecordForm(request.POST or None, request.FILES or None,instance=current_record)
+# 		if form.is_valid():
+# 			form.save()
+# 			messages.success(request, "Record Has Been Updated!")
+# 			return redirect('home')
+# 		return render(request, 'update_record.html', {'form':form})
+# 	else:
+# 		messages.success(request, "You Must Be Logged In...")
+# 		return redirect('home')
 
-def add_record(request):
-	form = AddRecordForm(request.POST or None)
-	if request.user.is_authenticated:
-		if request.method == "POST":
-			if form.is_valid():
-				add_record = form.save()
-				messages.success(request, "Record Added...")
-				return redirect('home')
-		return render(request, 'add_record.html', {'form':form})
-	else:
-		messages.success(request, "You Must Be Logged In...")
-		return redirect('home')
-def update_record(request, pk):
-	if request.user.is_authenticated:
-		current_record = Record.objects.get(id=pk)
-		form = AddRecordForm(request.POST or None, instance=current_record)
-		if form.is_valid():
-			form.save()
-			messages.success(request, "Record Has Been Updated!")
-			return redirect('home')
-		return render(request, 'update_record.html', {'form':form})
-	else:
-		messages.success(request, "You Must Be Logged In...")
-		return redirect('home')
\ No newline at end of file
+class update_record(LoginRequiredMixin,View):
```

Comment:
class name should be capitalize e.g `UpdateRecord`
snake case is for function/method/variable name

Reply:
i have Capitalize (Pascal cased ) all the names of class